### DISCUSSION TITLE
PHP Mailer 2.0.4 not verifying captcha image

### DISCUSSION THREAD

**Discussion by boattowers** [PHPMailer/PHPMailer#2649] [General]
Hello , I need some help with an older version of PHP Mailer 2.0.4
Mailer is not verifying the text entered by user with the one created by the script. 
No matter what is entered correct or incorrect we get the following msg..
"The code does not match. Please try again!" 
Can someone help please ?
Thank you

**Comment by Synchro** [comment]
Sorry, but PHPMailer has nothing to do with CAPTCHAs, and I'm not supporting a version of PHPMailer that's 15 years old. Refer to wherever your captcha script came from, and update before you get hacked.

**Reply by boattowers** [reply]
thanks for the help
